libraries/ortbConverter/processors/schain.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * This module handles the relocation of schain object from source.schain to source.ext.schain
+ */
+import {deepAccess, deepSetValue} from '../../../src/utils.js';
+import {REQUEST} from '../../../src/pbjsORTB.js';
+
+/**
+ * Processor to move schain from source.schain to source.ext.schain
+ * This ensures that the schain object is validated first and then moved to the ext object
+ */
+export const schainProcessor = {
+  [REQUEST]: {
```

Comment:
you're not using this structure, you could export just `moveSchainToExt`

Reply:
Thanks for pointing it out. Made the changes